games-fps/serious-sam-tse/serious-sam-tse-1_beta1-r1.ebuild

```diff
@@ -0,0 +1,137 @@
+# Copyright 1999-2023 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+inherit cdrom eutils versionator unpacker
+
+# MY_PV will be e.g. "beta1"
+MY_PN="ssamtse"
+MY_PV=$(get_version_component_range 2-2)
+
+DESCRIPTION="Serious Sam: The Second Encounter"
+HOMEPAGE="http://www.croteam.com/
+	http://www.seriouszone.com/"
+SRC_URI="http://icculus.org/betas/ssam/${MY_PN}-${MY_PV}.sh.bin"
```

Comment:
@ulm 
I made all the changes as you asked.
Including those listed above.
I deleted it http://www.seriouszone.com/ because the forum closed a few years ago.